hub/core/chunk/chunk_compressed_chunk.py

```diff
@@ -126,7 +129,15 @@ def extend_if_has_space_image_compression(
             num_samples += 1
         return num_samples
 
+    def _get_partial_sample_tile(self):
+        if self.decompressed_samples or self.decompressed_bytes:
+            return None
+        return super(ChunkCompressedChunk, self)._get_partial_sample_tile()
+
     def read_sample(self, local_index: int, cast: bool = True, copy: bool = False):
+        partial_sample_tile = self._get_partial_sample_tile()
+        if partial_sample_tile is not None:
+            return partial_sample_tile
```

Comment:
when would this happen?

Reply:
also codecov

Reply:
Happens when it is a partial sample tile, i.e not updated yet.